0.16.2 do not compile on MacOS

I've got this error when I'm trying to compile my project with sysinfo version 0.16.2 :

```
   Compiling sysinfo v0.16.2
error[E0425]: cannot find function `getfsstat` in crate `libc`
  --> /Users/cregal/.cargo/registry/src/github.com-1ecc6299db9ec823/sysinfo-0.16.2/src/apple/macos/disk.rs:44:32
   |
44 |     let count = unsafe { libc::getfsstat(ptr::null_mut(), 0, libc::MNT_NOWAIT) };
   |                                ^^^^^^^^^ not found in `libc`

error[E0425]: cannot find value `MNT_NOWAIT` in crate `libc`
    --> /Users/cregal/.cargo/registry/src/github.com-1ecc6299db9ec823/sysinfo-0.16.2/src/apple/macos/disk.rs:44:68
     |
44   |     let count = unsafe { libc::getfsstat(ptr::null_mut(), 0, libc::MNT_NOWAIT) };
     |                                                                    ^^^^^^^^^^ help: a constant with a similar name exists: `IPC_NOWAIT`
     |
    ::: /Users/cregal/.cargo/registry/src/github.com-1ecc6299db9ec823/libc-0.2.80/src/unix/bsd/apple/mod.rs:3103:1
     |
3103 | pub const IPC_NOWAIT: ::c_int = 0x800;
     | -------------------------------------- similarly named constant `IPC_NOWAIT` defined here

error[E0425]: cannot find function `getfsstat` in crate `libc`
  --> /Users/cregal/.cargo/registry/src/github.com-1ecc6299db9ec823/sysinfo-0.16.2/src/apple/macos/disk.rs:50:32
   |
50 |     let count = unsafe { libc::getfsstat(disks.as_mut_ptr(), bufsize, libc::MNT_NOWAIT) };
   |                                ^^^^^^^^^ not found in `libc`

error[E0425]: cannot find value `MNT_NOWAIT` in crate `libc`
    --> /Users/cregal/.cargo/registry/src/github.com-1ecc6299db9ec823/sysinfo-0.16.2/src/apple/macos/disk.rs:50:77
     |
50   |     let count = unsafe { libc::getfsstat(disks.as_mut_ptr(), bufsize, libc::MNT_NOWAIT) };
     |                                                                             ^^^^^^^^^^ help: a constant with a similar name exists: `IPC_NOWAIT`
     |
    ::: /Users/cregal/.cargo/registry/src/github.com-1ecc6299db9ec823/libc-0.2.80/src/unix/bsd/apple/mod.rs:3103:1
     |
3103 | pub const IPC_NOWAIT: ::c_int = 0x800;
     | -------------------------------------- similarly named constant `IPC_NOWAIT` defined here

error: aborting due to 4 previous errors

For more information about this error, try `rustc --explain E0425`.
error: could not compile `sysinfo`
```

OS Version : MacOS Catalina 10.15.7

It works with version `0.15.9`

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -24,7 +24,7 @@ winapi = { version = "0.3", features = ["fileapi", "handleapi", "ifdef", "ioapis
 ntapi = "0.3"
 
 [target.'cfg(not(any(target_os = "unknown", target_arch = "wasm32")))'.dependencies]
-libc = "0.2"
+libc = "^0.2.86"
 
 [target.'cfg(any(target_os = "macos", target_os = "ios"))'.dependencies]
 core-foundation-sys = "0.8"

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -1,6 +1,6 @@
 [package]
 name = "sysinfo"
-version = "0.16.2"
+version = "0.16.3"
 authors = ["Guillaume Gomez <guillaume1.gomez@gmail.com>"]
 
 description = "Library to get system information such as processes, processors, disks, components and networks"

